JSDemos/Demos/Form/Validation/jQuery/index.js

```diff
@@ -176,29 +194,64 @@ $(() => {
           message: 'The phone must have a correct USA phone format',
         }],
       }, {
-        dataField: 'Accepted',
-        label: {
-          visible: false,
-        },
-        editorOptions: {
-          text: 'I agree to the Terms and Conditions',
-        },
-        validationRules: [{
-          type: 'compare',
-          comparisonTarget() { return true; },
-          message: 'You must agree to the Terms and Conditions',
-        }],
-      }],
-    }, {
-      itemType: 'button',
-      horizontalAlignment: 'left',
-      buttonOptions: {
-        text: 'Register',
-        type: 'success',
-        useSubmitBehavior: true,
-      },
-    }],
-  }).dxForm('instance');
+          itemType: 'group',
+          cssClass: 'last-group',
+          colCountByScreen: {
+                  xs: 2, 
+                  sm: 2,
+                  md: 2,
+                  lg: 2
+              },
+          items: [{
+              dataField: 'Accepted',
+              label: {
+                visible: false,
+              },
+              editorOptions: {
+                text: 'I agree to the Terms and Conditions',
+              },
+              validationRules: [{
+                type: 'compare',
+                comparisonTarget() { return true; },
+                message: 'You must agree to the Terms and Conditions',
+              }],
+            }, 
+            {
+              itemType: 'group',
+              cssClass: 'buttons-group',
+              colCountByScreen: {
+                  xs: 2, 
+                  sm: 2,
+                  md: 2,
+                  lg: 2
+              },
+              items: [
+              {
+                  itemType: 'button',
+                  name: 'Reset',
+                  buttonOptions: {
+                      onClick: () => {
+                        formWidget.reset();
+                      },
+                      icon: 'refresh',
+                      text: 'Reset',
+                      disabled: true,
+                      width:'120px',
+
+                  },
+              }, {
+                  itemType: 'button',
+                  buttonOptions: {
+                      text: 'Register',
+                      type: 'default',
+                      useSubmitBehavior: true,
+                      width:'120px'
```

Comment:
U definitely have some issue with local linter
```suggestion
                      width: '120px'
```